### ISSUE TITLE
Project 3- Review My App #3

### ISSUE THREAD

**Issue by aboudreauxa** [aboudreauxa/Insuhgram#1]
My app, Insuhgram, is complete. Please review. /cc @codepathreview @codepath

**Comment by codepathreview** [comment]
Hi Alexina,

the gif is not working, and the link for the gif is not working. can you double check and comment on this issue to let us know when fixed?

**Comment by aboudreauxa** [comment]
Hello,

I am sorry I did not respond to this earlier. I have had a very busy weekend and didn't have time to check my emails until just now. I'm not sure why the gif didn't work, I copy and pasted the link in another tab to make sure that it was fine when I submitted it, but maybe it didn't work because I don't have an account or something. I have attached the gif to this post, but let me know if you would like me to do something else with it.

Thanks!
--Alexina

/cc @codepathreview @codepath

![insuhgramz](https://cloud.githubusercontent.com/assets/19497365/16372655/89a4f3bc-3c02-11e6-9808-27095586416e.gif)
